cmd/build.go

```diff
@@ -532,13 +533,17 @@ func buildNormal(targetOS string, vmArguments []string) {
 
 func buildEnv(targetOS string, engineCachePath string) []string {
 	var cgoLdflags string
+
+	outputDirPath := build.OutputDirectoryPath(targetOS)
+
 	switch targetOS {
 	case "darwin":
 		cgoLdflags = fmt.Sprintf("-F%s -Wl,-rpath,@executable_path", engineCachePath)
+		cgoLdflags = fmt.Sprintf("%s -F%s -L%s", cgoLdflags, outputDirPath, outputDirPath)
```

Comment:
Why are these flags removed: `-Wl,-rpath,@executable_path`? Are they not necessary anymore? Did you test?

Reply:
It's not removed, the changes adds -F and -L flags to the existing `cgoLdflags` ;)